<?xml version="1.0" encoding="UTF-8"?>
<launch>

  <!-- Camera parameters -->
  <arg name="rgb_image_topic" default="kinect2/qhd/image_color_rect"/>

  <arg name="output_data_directory" default="$(find uwds3)/data/objects"/>

  <arg name="label" default="unicorn"/>
  <arg name="max_samples" default="100"/>

  <node name="object_recorder" pkg="uwds3" type="object_recorder_node.py" args="" output="screen" respawn="false">
    <rosparam subst_value="true">

      rgb_image_topic: $(arg rgb_image_topic)

      output_data_directory: $(arg output_data_directory)
      label: $(arg label)

    </rosparam>
  </node>

</launch>
